JMJD6.py

```diff
@@ -0,0 +1,115 @@
+from abc import ABC, abstractmethod
+
+
+class BiologicalSequence(ABC):
+    @abstractmethod
+    def seq_len():
+        pass
+
+    def seq_slices():
+        pass
+
+    def seq_indexes():
+        pass
+
+    def seq_to_string():
+        pass
+
+    def check_alphabet():
+        pass

```

Comment:
- Лучше использовать dunder-методы. 
- Добавить декоратор `@abstractmethod` ко всем функциям, которые будут переписаны в дочерних классах
```suggestion
    @abstractmethod
    def __len__(self):
        pass

    @abstractmethod
    def __getitem__(self):
        pass

    @abstractmethod
    def __str__(self):
        pass

    @abstractmethod
    def is_alphabet_correct(self):
        pass
```